docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md

```diff
@@ -0,0 +1,29 @@
+# What collected IoT data can you find in the Coiote?
+
+The Coiote IoT Device Management Platform enables you to gather diverse data types through the standardized data model offered by the LwM2M protocol.
```

Comment:
`gather data through data model`? Maybe it's correct, I don't know, but from my point of view, `gather data inside data model` or something.